core/class-extension-core.php

```diff
@@ -305,22 +302,22 @@ function add_custom_events_path( $paths ) {
 		 *
 		 * @return string
 		 */
-		public function getExtentionPluginName() {
-			if ( null === $this->extention_plugin_name ) {
-				$this->extention_plugin_name = '';
+		public function get_extention_plugin_name() {
```

Comment:
Let's also fix the function name to `get_extension_plugin_name`.
```suggestion
		public function get_extension_plugin_name() {
```